packages/next-templates/src/app/globals.css

```diff
@@ -20,7 +20,8 @@ body {
 
 .voorbeeld-theme {
   /* HACK: these tokens are not set in voorbeeld-theme */
-  --utrecht-form-field-description-margin-block-end: var(--voorbeeld-space-block-beetle);
+  --voorbeeld-focus-background-color: --utrecht-form-field-description-margin-block-end:
```

Comment:
🤔 Dit kan toch nooit werken? Denk dat hier met rebasen misschien iets mis is gegaan.